devon4ng-mat-layout/index.asciidoc

```diff
@@ -0,0 +1,27 @@
+= devon4ng Angular Material layout
+====
+The purpose of this tutorial is to get a basic understanding of creating layouts using Angular Material in a devon4ng application. We will create an application with a header containing some menu links and a sidenav with some navigation links.
+====
+
+[step]
+--
+restoreDevonfwIde(["java","mvn", "npm", "ng"])
+--
+====
+Before creating a devon4ng application, you first have to install the devonfw ide. You will find more information about devonfw on https://devonfw.com/website/pages/welcome/welcome.html.
+Once you have setup devonfw ide, you are ready to create your first devon4ng project.
+[step]
+--
+createDevon4ngProject("devon4ng-mat-layout", "")
+--
+Enter
+
+`y` {{execute T1}} 
+
+when it asks whether it would like to add Angular routing and select `SCSS` when it asks for the stylesheet format.
```

Comment:
You can also pass parameters to createDevon4ngProject as described here: https://github.com/devonfw-tutorials/tutorials/wiki/Functions
So you can simply write createDevon4ngProject("devon4ng-mat-layout", ["--style=scss"])